cmd/goProbe/flags/root.go

```diff
@@ -0,0 +1,36 @@
+// Package flags is for parsing goProbe's command line parameters.
+package flags
+
+import (
+	"errors"
+	"flag"
+)
+
+// Flags stores goProbe's command line parameters
+type Flags struct {
+	Config  string
+	DocGen  bool
+	Version bool
+}
+
+// CmdLine globally exposes the parsed flags
+var CmdLine *Flags
+
+func init() {
+	CmdLine = &Flags{}
```

Comment:
Can't you just use var CmdLine = new(Flags) above instead of using the init() functionality?